docker-compose.test.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.6'
```

Comment:
Don't think this file is needed.

Reply:
I have added this to make life of testing easy, when using database.

Reply:
Ah, I usually just use a single docker compose for development. What does `hms` mean in this context?

Reply:
renamed to `app` and added `readme.md`